hadoop-ozone/dist/src/main/compose/ozonesecure-ha/om-bootstrap.yaml

```diff
@@ -0,0 +1,73 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3.8"
+
+x-OM-config:
+  &common-env
+  env_file:
+    - ./docker-config
+    - ./docker-config-om-ratis
+
+services:
+  kdc:
+    <<: *common-env
+  kms:
+    <<: *common-env
+  datanode1:
+    <<: *common-env
+  datanode2:
+    <<: *common-env
+  datanode3:
+    <<: *common-env
+  om1:
+    <<: *common-env
+  om2:
+    <<: *common-env
+  om3:
```

Comment:
These all override the original settings for om3 in docker-compose.yaml, right?  since they are mostly the same as the ones there, we don't need them do we?  Can we just have the parts that are different, something like this:
```
  om3:
    # This command produces the same behavior as sleeping indefinitely.
    command: ["tail","-f","/dev/null"]
    <<: *common-env
```

Reply:
We are also skipping the initialisation part, so that the container starts empty and sleeps forever. A simple sleep won't do it, if the container has been initialised.

```diff
    environment:
-      WAITFOR: scm3.org:9894
-      ENSURE_OM_INITIALIZED: /data/metadata/om/current/VERSION
      OZONE_OPTS:

```

Check [here](https://github.com/xBis7/ozone/blob/HDDS-9266/hadoop-ozone/dist/src/main/compose/ozonesecure-ha/docker-compose.yaml#L173-L174).

> Can we just have the parts that are different, something like this:

Yes, I think it can work like so

1. config variable for om3
    ```
    x-om3-setup:
      &om3-setup
       environment:
          OZONE_OPTS:
       # This command produces the same behavior as sleeping indefinitely.
       command: ["tail","-f","/dev/null"] 
    ```

2. and then om3 service will be
    ```
    om3:
      <<: *om3-setup 
      <<: *common-env
    ```

Reply:
With the above approach
```
yaml: map merge requires map or sequence of maps as the value
```